include/bluetooth/hci.h

```diff
@@ -2173,6 +2250,7 @@ struct bt_hci_evt_le_biginfo_adv_report {
 #define BT_EVT_MASK_LE_ADV_SET_TERMINATED        BT_EVT_BIT(17)
 #define BT_EVT_MASK_LE_SCAN_REQ_RECEIVED         BT_EVT_BIT(18)
 #define BT_EVT_MASK_LE_CHAN_SEL_ALGO             BT_EVT_BIT(19)
+#define BT_EVT_MASK_LE_PAST_RECEIVED    BT_EVT_BIT(23)
```

Comment:
There seems to be some alignment issue here?

Reply:
Yup. Did a bulk rename and must've missed the indentation here. Thanks!